qp_woltka/tests/test_woltka.py

```diff
@@ -181,8 +181,8 @@ def test_woltka_to_array_rep82(self):
             f'#SBATCH --job-name merge-{job_id}\n',
             '#SBATCH -N 1\n',
             '#SBATCH -n 5\n',
-            '#SBATCH --time 10:00:00\n',
-            '#SBATCH --mem 48g\n',
+            '#SBATCH --time 30:00:00\n',
+            '#SBATCH --mem 140g\n',
```

Comment:
@antgonza 48GB to 140GB seems like a big jump compared to the other change. I'm going to assume that's intentional, however.